tensorflow/core/kernels/data/range_dataset_op_test.cc

```diff
@@ -49,51 +51,54 @@ struct TestCase {
 };
 
 TestCase PositiveStepTestCase() {
-  return {/*start*/ 0,
-          /*stop*/ 10,
-          /*step*/ 3,
-          /*expected_outputs*/
+  return {/*start=*/0,
+          /*stop=*/10,
+          /*step=*/3,
+          /*expected_stateful=*/false,
+          /*expected_outputs=*/
           {DatasetOpsTestBase::CreateTensor<int64>(TensorShape({}), {0}),
            DatasetOpsTestBase::CreateTensor<int64>(TensorShape({}), {3}),
            DatasetOpsTestBase::CreateTensor<int64>(TensorShape({}), {6}),
            DatasetOpsTestBase::CreateTensor<int64>(TensorShape({}), {9})},
-          /*expected_output_dtypes*/ {DT_INT64},
-          /*expected_output_shapes*/ {PartialTensorShape({})},
-          /*expected_cardinality*/ 4,
-          /*breakpoints*/ {0, 1, 4}};
+          /*expected_output_dtypes=*/{DT_INT64},
+          /*expected_output_shapes=*/{PartialTensorShape({})},
+          /*expected_cardinality=*/4,
+          /*breakpoints=*/{0, 1, 4}};
 }
 
 TestCase NegativeStepTestCase() {
-  return {/*start*/ 10,
-          /*stop*/ 0,
-          /*step*/ -3,
-          /*expected_outputs*/
+  return {/*start=*/10,
+          /*stop=*/0,
+          /*step=*/-3,
+          /*expected_stateful=*/false,
+          /*expected_outputs=*/
           {DatasetOpsTestBase::CreateTensor<int64>(TensorShape({}), {10}),
            DatasetOpsTestBase::CreateTensor<int64>(TensorShape({}), {7}),
            DatasetOpsTestBase::CreateTensor<int64>(TensorShape({}), {4}),
            DatasetOpsTestBase::CreateTensor<int64>(TensorShape({}), {1})},
-          /*expected_output_dtypes*/ {DT_INT64},
-          /*expected_output_shapes*/ {PartialTensorShape({})},
-          /*expected_cardinality*/ 4,
-          /*breakpoints*/ {0, 1, 4}};
+          /*expected_output_dtypes=*/{DT_INT64},
+          /*expected_output_shapes=*/{PartialTensorShape({})},
+          /*expected_cardinality=*/4,
+          /*breakpoints=*/{0, 1, 4}};
 }
 
 TestCase ZeroStepTestCase() {
   return {/*start*/ 0,
           /*stop*/ 10,
           /*step*/ 0,
-          /*expected_outputs*/ {},
-          /*expected_output_dtypes*/ {},
-          /*expected_output_shapes*/ {},
-          /*expected_cardinality*/ 0,
-          /*breakpoints*/ {}};
+          /*expected_stateful=*/false,
+          /*expected_outputs=*/{},
+          /*expected_output_dtypes=*/{},
+          /*expected_output_shapes=*/{},
+          /*expected_cardinality=*/0,
+          /*breakpoints=*/{}};
 }
 
 class ParameterizedRangeDatasetOpTest
     : public RangeDatasetOpTest,
       public ::testing::WithParamInterface<TestCase> {};
 
-TEST_P(ParameterizedRangeDatasetOpTest, GetNext) {
+TEST_P(ParameterizedRangeDatasetOpTest, APIs) {
```

Comment:
There should (continue to) be separate unit tests for different API endpoints.

Reply:
Also, most of the API endpoints do not need to be tested with two test cases (if both test cases exercise the same code path).

Reply:
> There should (continue to) be separate unit tests for different API endpoints.

Ok, will change them back.